app/javascript/data/lockReasonOptions.js

```diff
@@ -0,0 +1,6 @@
+export const lockReasonOptions = [
+    'Not specified',
```

Comment:
a little nit picky, but we usually use 2 spaces for indentation

Reply:
Fixed! See [4e4e4f01e38b0d5d7dc0d80552cf7d1843a8c4ac](https://github.com/SaraAlert/SaraAlert/pull/1180/commits/4e4e4f01e38b0d5d7dc0d80552cf7d1843a8c4ac)